org.civicrm.bootstrapcivicrm/scss/bootstrap/mixins/_fa-icon.scss

```diff
@@ -1,17 +1,18 @@
-@mixin fa-icon($font-size: null, $content: null, $color: null) {
+@mixin fa-icon($args...) {
```

Comment:
I know what this change does, but I'm curious if this solve anything or is a plain code improvement?